server/pbenchinacan/run-pbench-in-a-can

```diff
@@ -1,5 +1,25 @@
 #!/bin/bash
 
+#
+# Specification for a self-contained Pbench Server Environment.
+#
+# This configuration creates a pod for the Pbench Server deployment using
+# four separate containers: the Pbench Server and its associated
+# Elasticsearch instance, PostgreSQL database instance, and KeyCloak broker
+# instance. It then runs the server/pbenchinacan/load_keycloak.sh to
+# configure the keycloak broker for the minimum requirements of the
+# Pbench-Server.
```

Comment:
You should probably recast this slightly:  "After starting the dependent services, this script configures the Keycloak broker, and then starts the Pbench Server."  We don't need to reference the name of the KC configuration script here (that will be in the code, and if it changes there we don't want to have to also change it here!), and we want to point out something about the sequencing (to wit, that we defer starting the Pbench Server).